ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/block/BlockProcessorElectra.java

```diff
@@ -725,16 +726,25 @@ private Optional<OperationInvalidReason> checkCommittees(
       final BeaconState state,
       final UInt64 slot,
       final SszBitlist aggregationBits) {
-    int participantsCount = 0;
+    int committeeOffset = 0;
     for (final UInt64 committeeIndex : committeeIndices) {
       if (committeeIndex.isGreaterThanOrEqualTo(committeeCountPerSlot)) {
         return Optional.of(AttestationInvalidReason.COMMITTEE_INDEX_TOO_HIGH);
       }
       final IntList committee =
           beaconStateAccessorsElectra.getBeaconCommittee(state, slot, committeeIndex);
-      participantsCount += committee.size();
+      final int currentCommitteeOffset = committeeOffset;
+      final boolean hasAtLeastOneParticipant =
```

Comment:
Looking here again, maybe is better to resemble the `committee_attesters` variable and we could call this `committeeHasAtLeastOneAttester`